doc/src/manual/variables-and-scoping.md

```diff
@@ -410,7 +410,7 @@ methods happen to exist cannot be used to determine the scopes of variables. Wit
 behavior, it's especially concerning that someone might have written the `for` loop first, had it
 working just fine, but later when someone else adds a new global far away—possibly in a different
 file—the code suddenly changes meaning and either breaks noisily or, worse still, silently does the
-wrong thing. This kind of "spooky action at a distance" is something that good programming language
+wrong thing. This kind of ["spooky action at a distance"](https://en.wikipedia.org/wiki/Action_at_a_distance) is something that good programming language
```

Comment:
Should probably link to the article about the concept in programming (https://en.wikipedia.org/wiki/Action_at_a_distance_(computer_programming)) instead of the one in phsysics

Reply:
Yep, you're right. Just updated it